### ISSUE TITLE
Use with Tomcat & Warble?

### ISSUE THREAD

**Issue by will-sterling** [mrbrdo/theine#15]
Can Theine be used in a Jruby on Rails application that is package for deployment with warble?

**Comment by mrbrdo** [comment]
I don't think that would work in the current state, but I'm not sure why you'd want to use it like that?
